### ISSUE TITLE
Add password to game options

### ISSUE THREAD

**Issue by MGageMorgan** [devgianlu/PYX-Reloaded#62]
This is a pretty obvious one - but only after people who use this remember that they don't want randos to join their game lol

**Comment by devgianlu** [comment]
Duplicate of #55

**Comment by MGageMorgan** [comment]
Oops shit sorry...didn't realize that was already in the works 😂😂
